corehq/apps/case_importer/static/case_importer/js/main.js

```diff
@@ -26,6 +26,19 @@ hqDefine("case_importer/js/main", [
         _.delay(function () {
             recentUploads.goToPage(1);
         });
+
+        $('input:file').change(function () {
+            const fileName = $(this).val();
+            if (fileName) {
+                $(':submit').attr('disabled', false).removeClass('disabled');
+            } else {
+                $(':submit').attr('disabled', true).addClass('disabled');
```

Comment:
You can also use `$(':submit').disableButtonNoSpinner()` for readability (and `enableButton` above), but it's the same code.

Reply:
Ah, didn't find those in a brief look around. Updated to use them instead.